code-challenges.test.js

```diff
@@ -10,53 +10,125 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// const numArr1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numArr2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
 
+// it("takes in an array of numbers and returns an array with all the numbers multiplied by 3", () => {
+//     const numArr1 = [6, 7, 8, 9, 10]
+//     const numArr2 = [24, 27, 30, 33, 36]
+//      expect(multBy3(numArr1)).toEqual([18, 21, 24, 27, 30])
+//      expect(multBy3(numArr2)).toEqual([72, 81, 90, 99, 108])
+//    })
+ 
+// const multBy3 = (array) => {
+//    return array.map(value => value * 3)
+// } 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("numberDivisibleBy3", () => {
+    it("takes a number as an argument and decides if the number is evenly divisible by three or not.", () => {
+        expect(numberDivisibleBy3("is divisible by three")).toEqual("is not divisible by three")
```

Comment:
I only see one expect method in this test, typically you will want to test all three given test variables.  Keep in mind, you will often want to place the variables before the expect statement and pass it into the function on line 43.